.github/CONTRIBUTING.md

```diff
@@ -125,15 +124,67 @@ For names made of multiple concatenated words, the first letter of the whole is
 For **C++ files** (*.cpp .h*): 4 spaces  
 For **Qt-UI files** (*.ui*): 2 spaces
 
-#### Pointers
+#### Includes
+```c
+// Class includes
+#include "MyWidget.h"
+#include "ui_MyWidget.h"
+
+// Global includes
+#include <QWidget>
+#include <stdin>
+
+// Application includes
+#include "core/Config.h"
+#include "core/FilePath.h"
```

Comment:
I actually prefer app includes before framework and global includes. So include order is from most to least specific.